infrastructure/labelmaker/labeltester.jinja

```diff
@@ -0,0 +1,90 @@
+{#
+Copyright 2017 Google Inc. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+#}
+
+{% set COMPUTE_URL_BASE = 'https://www.googleapis.com/compute/v1/' %}
+{% set ID = env['deployment'] + '-' + env['name'] %}
+
+resources:
+- name: {{ ID }}
+  type: compute.v1.instance
+  properties:
+    zone: {{ properties['zone'] }}
+    machineType: {{ properties["machineType"] }}
+    metadata:
+      items:
+        - key: startup-script
+          value: |
+            #!/bin/bash -e
+            apt-get update
+            apt-get -y install python-pip
+            pip install --upgrade google-api-python-client
+            gsutil cp {{ properties['labelmaker'] }} .
```

Comment:
Have you considered include the script in the deployment manager template from a yaml import?

It's a little known trick in deployment manager files:

in the yaml have: 

imports:
- path: label_maker.py

Then within the jinja file:

            cat > label_maker.py <<EOF
            {{ imports["label_maker.py"]|indent(12) }}
            EOF
            chmod +x label_maker.py
            ./label_maker.py /tmp/labels.json

This removes a couple steps for users looking to try the script out (no need to create bucket, upload object, edit yaml with url). 

Thanks